io.github.wxmaxima_developers.wxMaxima.yml

```diff
@@ -0,0 +1,75 @@
+app-id: io.github.wxmaxima_developers.wxMaxima
+runtime: org.freedesktop.Platform
+runtime-version: "23.08"
+sdk: org.freedesktop.Sdk
+command: wxmaxima
+finish-args:
+      - --share=ipc
+      - --socket=x11
+      - --env=LC_CTYPE="en_US.UTF-8"
```

Comment:
Why is this needed?

Reply:
As for the LC_CTYPE environment variable, it prevents a warning message from being displayed when executing a .mac file that prints to the screen.

Reply:
that also mean it will display numbers in a weird way for most of the world.

Reply:
Is there a way to pass the user's environment variable to wxMaxima?